src/interfaces/records/handlers/records-delete.ts

```diff
@@ -1,8 +1,9 @@
-import type { EventLog } from '../../../event-log/event-log.js';
+import type { DidResolver } from '../../../index.js';
 import type { MethodHandler } from '../../types.js';
 import type { RecordsDeleteMessage } from '../types.js';
+import type { StorageController } from '../../../store/storage-controller.js';
 import type { TimestampedMessage } from '../../../core/types.js';
-import type { DataStore, DidResolver, MessageStore } from '../../../index.js';
+
```

Comment:
unnecessary newline

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/42f36fb0385704079ff7f0a11a32f953d80a25c7